conf/evolutions/collins/16.sql

```diff
@@ -0,0 +1,9 @@
+# --- Add logical name of disks to asset meta
+
+# --- !Ups
+
+INSERT INTO asset_meta (name, priority, label, description) VALUES ('DISK_LOGICAL_NAME', -1, 'Disk logical name', 'The logical name of the disk as reported by lshw');
```

Comment:
Not sure if this is strictly needed since we're using dynamically created asset meta, but the tests freaked out when not adding it explicitly.